<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransactionListDialog</class>
 <widget class="QDialog" name="TransactionListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Previous Transaction</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="lineNumber">
     <property name="placeholderText">
      <string>Search transaction number</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LibGUI::TableWidget" name="tableWidget" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font-size: 12px;</string>
        </property>
        <property name="text">
         <string>p : print transaction</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LibGUI::TableWidget</class>
   <extends>QWidget</extends>
   <header>tablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
